jetty-core/jetty-server/pom.xml

```diff
@@ -59,6 +59,12 @@
       <artifactId>jetty-xml</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
If the "raw" test class is removed, this can go too.

Reply:
Done.